### DISCUSSION TITLE
HW Tone Mapping - Advice

### DISCUSSION THREAD

**Discussion by frdbonif** [jellyfin/jellyfin#5158] [Q&A]
Hello,

I was recently reading the PRs relating to tone mapping and that it is now available for both AMD and Nvidia discrete cards.

I'm after an AMD Card that will support tone mapping 4k video, I was looking for an RX460 but wondered if it will be enough - if not, can someone recommend an alternative card - ideally AMD but I won't write off Nvidia completely, I just have much more experience with AMD running on Linux.

I can't imagine I need many simultaneous transcodes, maybe two to four HD-SD transcodes and the occasional 4K tone mapping transcode.

Thanks for any help
Fred

**Comment by hurricane-dorian** [comment]
https://www.reddit.com/r/jellyfin/comments/l9doay/is_an_amd_apu_good_enough_for_4k_hevc_hdrsdr/ is worth a read. 
The gist of it is that ffmpeg is not very good at openCL tonemapping. If you have an Intel processors, QuickSync is way more efficient.
